thermalnetwork/network.py

```diff
@@ -674,9 +674,13 @@ def update_sys_params(self, system_parameter_path: Path, output_directory_path:
                 if ghe_sys_params["ghe_id"] == ghe_id:
                     # Update system parameters dict with the new values
                     bh_len = ghe_data["active_borehole_length"]["value"]
-                    num_boreholes = ghe_data["number_of_boreholes"]
-                    ghe_sys_params["borehole"]["length_of_boreholes"] = bh_len
-                    ghe_sys_params["borehole"]["number_of_boreholes"] = num_boreholes
+                    if ghe_sys_params["borehole"].get("number_of_boreholes_autosized"):
+                        num_boreholes = ghe_data["number_of_boreholes"]
+                        ghe_sys_params["borehole"]["number_of_boreholes"] = num_boreholes
+                    else:
```

Comment:
GHED still calculates number of boreholes, we are now ignoring that output. That doesn't feel right. We don't pass any of these values to GHED (AFAICT). Is this really the correct way to handle manual sizing?

Reply:
We're going to need a way for users to input the GHE config. Without adding a bunch of worker methods inside for square, rectangular, etc, we should just need to sets of data: the bh length and xy coordinate pairs of the borehole locations. We're working on ingesting that into GHED so we can the calculate the g-functions values. I think the rest of the needed info is already there.

Reply:
If that is additional fields in sys-param file (and it's ready), we can add it to https://github.com/urbanopt/geojson-modelica-translator/pull/712